### PR TITLE
Add typo variations for 'inherit'

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -16077,6 +16077,11 @@ inegrated->integrated
 ineqality->inequality
 inequalitiy->inequality
 inerface->interface
+inerit->inherit
+ineritance->inheritance
+inerited->inherited
+ineriting->inheriting
+inerits->inherits
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:16084]: Add `ineritor` & `ineritors`?
```suggestion
inerit->inherit
ineritance->inheritance
inerited->inherited
ineriting->inheriting
inerits->inherits
ineritor->inheritor
ineritors->inheritors
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -16077,6 +16077,13 @@ inegrated->integrated
...
+ineritor->inheritor
+ineritors->inheritors
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:16086]:
```suggestion
inerit->inherit
ineritance->inheritance
inerited->inherited
ineriting->inheriting
ineritor->inheritor
ineritors->inheritors
inerits->inherits
```

**Review by bl-ue — COMMENTED** [review]
Ooops!

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
@luzpaz, you were invited to join the organization. https://github.com/codespell-project

**Comment by luzpaz** [comment]
Thanks @bl-ue, but i see no invitation there

**Pull request merged**